### ISSUE TITLE
enhancement: hide votes on chat

### ISSUE THREAD

**Issue by Rocavall** [springweblobby/swl-website#40]
Since there is already a bar that shows votes, it might be a good idea to hide redundant information, such as "!vote y/n" commands issued by players and vote info displayed by springie "Hydrogen Poll: Start game? [!y=2/4, !n=0/4]"
"Hydrogen Poll: Start game? [END:SUCCESS]" Should be still displayed though.
What each player votes could be displayed by some tick/cross icon next to nickname in the playerlist when voting is happening.

**Comment by Rocavall** [comment]
In rooms with a lot of players votespam becomes pretty annoying

![kloot4](https://cloud.githubusercontent.com/assets/7760018/10418381/6519bf20-7059-11e5-8a13-34b3a7913678.png)

**Comment by db81** [comment]
Yeah, I do intend to hide them.

**Comment by lumiera** [comment]
But that way people will not know who voted and how. Please consider leaving this visible an option.

**Comment by db81** [comment]
It will probably show who voted how in the player list.

**Comment by AppSynergy** [comment]
How about:

+ the option to hide *(all / some / just the vote)* type 1 messages (from the server, shown in yellow). How many are really necessary?

The tick / cross idea seems great too, either way.

**Comment by db81** [comment]
Yeah it would be good to have extra filtering for stuff like ```Hi ikinz (rights:2), welcome to Springie 1.3.2.118, automated host. For help say !help```. I don't think it's a good idea to filter all BattlePrivate messages period.
